MaxText/configs/base.yml

```diff
@@ -271,9 +271,17 @@ inference_microbenchmark_prefill_lengths: "64,128,256,512,1024"
 inference_microbenchmark_stages: "prefill,generate"
 inference_microbenchmark_loop_iters: 10
 inference_microbenchmark_log_file_path: ""
+inference_microbenchmark_flatten_results: False
 
 key_value_axis_order_product_id: ""
 prefill_key_axis_order: "1,2,0,3"
 prefill_value_axis_order: "1,2,0,3"
 ar_key_axis_order: "1,2,0,3"
-ar_value_axis_order: "1,2,0,3"
\ No newline at end of file
+ar_value_axis_order: "1,2,0,3"
+
+# Lists of KV-related settings over which to loop.
+# Each iteration will take same index from every list; therefore lists must be of the same length
+inference_microbenchmark_sweep_additional_metadata: ""
+inference_microbenchmark_sweep_key_value_axis_order_product_id_list: ""
+inference_microbenchmark_sweep_ar_key_axis_order_list: ""
+inference_microbenchmark_sweep_ar_value_axis_order_list: ""
```

Comment:
These seems save the same purpose as
```
key_value_axis_order_product_id: ""
prefill_key_axis_order: "1,2,0,3"
prefill_value_axis_order: "1,2,0,3"
ar_key_axis_order: "1,2,0,3"
ar_value_axis_order: "1,2,0,3"
```

Reply:
The difference is that the `inference_microbenchmark_sweep_ar_key_axis_order_list` contains a colon `:` delimited list of the different things to try
```
'0,1,2,3:0,1,3,2:0,2,1,3:0,2,3,1'
```
Each VM will have a list of these that it will sweep across. It was the solution that was sort of the easiest to implement at the time.